### PR TITLE
feat: override Input and Button style components

### PR DESCRIPTION
**Pull request by Siqill**
### Type of change:

- [ ] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [x] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist:

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible
> If not explain here
- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change
Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

#### Screenshots/Video after change
Add screenshots with the changes applied here. 

buttons ark
buttons light

### CODE REVIEW

File: src/pages/Landing.jsx [hunk 1]
```diff
@@ -1,15 +1,44 @@
-import { Box, Flex } from '@chakra-ui/react';
+import { Box, Button, Flex, Input, Text } from '@chakra-ui/react';
 
 import { AddWallet, Footer } from '@/components';
 
+// import i18next from 'i18next';
+// import { Footer } from '@/components';
```
Comment (kalashnikau) [inline src/pages/Landing.jsx:6]: why its commented? remove commented lines or uncomment them

**Comment by lazzzaroni** [comment]
I don't think that it's accessible colors for input field... :)
You simply can't read info in placeholders when field is active.

https://user-images.githubusercontent.com/68731150/180180382-1cb64fc9-450f-4507-9400-c2d3cbd09e14.mov

And consider to leave i18next, because you'll need to use it for every English word in our workflow.
Or share your opinion why do you think that you don't need i18next in your changes?

**Comment by Siqill** [comment]
> I don't think that it's accessible colors for input field... :) You simply can't read info in placeholders when field is active.
> 
>  Screen.Recording.2022-07-21.at.11.24.56.mov 
> And consider to leave i18next, because you'll need to use it for every English word in our workflow. Or share your opinion why do you think that you don't need i18next in your changes?

First of all i took those colours from the ready-made preset of colors.
Secondly the main bg colour must toggle as well with dark/light mode and then its look good as for me
Probably there is wrong bg colour cos it must be orange.200(not besqie) and then if looks better

What about i18next ill fix it and push text into it
image

**Review by Yulatun — APPROVED** [review]

**Comment by Siqill** [comment]
image

image

**Review by kalashnikau — APPROVED** [review]

**Review by itsnotpolza — APPROVED** [review]

**Pull request merged**